tests/flow/constraint_utils.py

```
@@ -0,0 +1,152 @@
+
+def create_mandatory_constraint(g, entity_type, lbl, *props, sync=False):
+    return create_constraint(g, "MANDATORY", entity_type, lbl, *props, sync=sync)
+
+def create_unique_node_constraint(g, lbl, *props, sync=False):
+    # create exact-match index(es)
+    for p in props:
+        # OK to fail if index already exists
+        try:
+            create_node_range_index(g, lbl, p, sync=False)
+        except:
+            pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)